dino_catalog/dinosaur.rb

```diff
@@ -0,0 +1,62 @@
+class Dinosaur
+  attr_accessor :name, :period, :continent, :diet, :weight_in_lbs, :walking, :description
+
+  def initialize(args={})
+    @name = args[:name]
+    @period = args[:period]
+    @continent = args[:continent]
+    @diet = args[:diet]
+    @weight_in_lbs = args[:weight_in_lbs]
+    @walking = args[:walking]
+    @description = args[:description]
+
+    validate_weight!(@weight_in_lbs)
+    validate_description!(@description)
```

Comment:
Validation only happens in the `initialize` but the fields are writable so this model doesn't really guarantee validness.